### PR TITLE
myeid: support logout operation

### PR DESCRIPTION
**Pull request by fabled**
cc @hhonkanen

### CODE REVIEW

File: src/libopensc/card-myeid.c [hunk 1]
```diff
@@ -427,6 +427,21 @@ static int myeid_select_file(struct sc_card *card, const struct sc_path *in_path
...
 
+static int myeid_logout(struct sc_card *card)
+{
+	sc_apdu_t apdu;
+	int r;
+
+	LOG_FUNC_CALLED(card->ctx);
+
+	sc_format_apdu(card, &apdu, SC_APDU_CASE_1, 0x2E, 0x00, 0x00 /*pin ref: 0x01-0x0E, 0=ALL*/);
+	apdu.cla    = 0;
```
Comment (Jakuje) [inline src/libopensc/card-myeid.c:438]:
```suggestion
	apdu.cla = 0;
```

File: src/libopensc/card-myeid.c [hunk 2]
```diff
@@ -427,6 +427,21 @@ static int myeid_select_file(struct sc_card *card, const struct sc_path *in_path
...
+
+	r = sc_transmit_apdu(card, &apdu);
+	LOG_TEST_RET(card->ctx, r,  "APDU transmit failed");
```
Comment (Jakuje) [inline src/libopensc/card-myeid.c:441]:
```suggestion
	LOG_TEST_RET(card->ctx, r, "APDU transmit failed");
```

**Review by Jakuje — COMMENTED** [review]
there are minor formatting problems, but otherwise I think it looks ok.

**Comment by hhonkanen** [comment]
This looks ok.

**Review by Jakuje — APPROVED** [review]

**Comment by Jakuje** [comment]
Thank you! Merged

**Pull request merged**